src/llm_vm/onsite_llm.py

```diff
@@ -296,12 +296,12 @@ class Small_Local_LLama(Base_Onsite_LLM):
         tokenizer_loader: Loads the tokenizer into memory
         generate: Generates a response from a given prompt with the loaded LLM and tokenizer
     """
-    model_uri="openlm-research/open_llama_3b_v2"
+    model_uri="meta-llama/Llama-2-70b-hf"
```

Comment:
changes this to 7B or whatever its called and i'll merge it in

Reply:
I have downgrade the version to the lowest, as here mentioned: https://huggingface.co/meta-llama

Reply:
70b is much bigger than 7b

Reply:
woops nvm